images/origin/system-container/config.json.template

```diff
@@ -70,15 +70,13 @@
 	    "source": "proc"
 	},
 	{
-	    "destination": "/dev",
-	    "type": "tmpfs",
-	    "source": "tmpfs",
-	    "options": [
-		"nosuid",
-		"strictatime",
-		"mode=755",
-		"size=65536k"
-	    ]
+		"destination": "/dev",
+		"type": "bind",
+		"source": "/dev",
+		"options": [
+			"rbind",
+			"rprivate"
```

Comment:
Will it still work if we use rslave instead of rprivate? We will make sure any change on the host is propagated to the container (even if /dev most probably won't change)